src/Progress/Progress.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useEffect, useRef, useState, forwardRef, memo} from 'react'
+import {View, StyleSheet, Animated, Dimensions} from 'react-native'
+
+interface IProgressProps {
+  /**
+   * Current percent of the Progress
+   * default 0, max 100
+   */
+  value: number
+  /**
+   * Defines height of Progress
+   * default 16
+   */
+  size?: number
+  /**
+   * Defines borderRadius of Progress
+   * default 0
+   */
+  borderRadius?: number
+  /**
+   * Defines color of Track Bar
+   * default #49BE25
+   */
+  filledTrackColor?: string
+  /**
+   * Defines background color of Progress
+   * default #E5E5E5
+   */
+  backgroundColor?: string
+}
+
+const {width: screenWidth} = Dimensions.get('screen')
+
+const Progress = (props: IProgressProps, ref?: any) => {
+  const {
+    value = 0,
+    size = 16,
```

Comment:
use `metrics` we defined in `theme`

Reply:
Will update when merge branch, because other member already added the theme folder

Reply:
Okay, skip for now